src/cronjobs/tokenService.ts

```diff
@@ -32,51 +32,123 @@ async function insertTokensToDatabase(ctx: CronJob) {
       const chainId = chain.chainId;
       if (!chainId) {
         logger.error('chainId is required');
-        continue; // Skip this iteration if chainId is missing
+        continue;
       }
 
       const fetchedTokens = await fetchTokensByChain(chainId);
-
-      // Ensure fetchedTokens is an array
       if (!Array.isArray(fetchedTokens) || fetchedTokens.length === 0) {
         logger.warn(`No tokens found for chainId ${chainId}`);
-        continue; // Skip if no tokens fetched
+        continue;
       }
 
-      const values = await Promise.all(
-        fetchedTokens.map(async (token: any) => ({
+      let dbTokens = [];
+      try {
+        dbTokens = await db
+          .select()
+          .from(tokens)
+          .where(sql`${tokens.chainId} = ${chainId}`)
```

Comment:
`eq` is imported from `drizzle-orm`, not from lodash.

```suggestion
          .where(eq(tokens.chainId, chainId))
```